pymatgen/phonon/plotter.py

```diff
@@ -407,6 +548,46 @@ def save_plot(self, filename, img_format="eps", ylim=None, units="thz"):
         plt.savefig(filename, format=img_format)
         plt.close()
 
+    def show_proj(
+        self,
+        site_comb: str | list[list[int]] = "element",
+        ylim: tuple[None | float, None | float] = None,
+        units: str = "thz",
+        rgb_labels: tuple[str] = None,
+    ):
+        """
+        Show the projected plot using matplotlib.
+
+        Args:
+            ylim: Specify the y-axis (frequency) limits; by default None let
+                the code choose.
+            units: units for the frequencies. Accepted values thz, ev, mev, ha, cm-1, cm^-1.
+        """
+        self.get_proj_plot(site_comb=site_comb, ylim=ylim, units=units, rgb_labels=rgb_labels)
+        plt.show()
+
+    def save_proj_plot(
+        self,
+        filename,
+        img_format="eps",
+        site_comb: str | list[list[int]] = "element",
+        ylim: tuple[None | float, None | float] = None,
+        units: str = "thz",
+        rgb_labels: tuple[str] = None,
+    ):
+        """
+        Save matplotlib projected plot to a file.
```

Comment:
One more question: do we really need `save_proj_plot()`? Seems like it's a wrapper around calling `fig.savefig()` which can just be left to the user.

Reply:
> I think you force-pushed over the commit [709cda1](https://github.com/materialsproject/pymatgen/commit/709cda100218e47dba247ba90aadf649e1d7912b) I added after [702b5bd](https://github.com/materialsproject/pymatgen/commit/702b5bdc8e054b0f269dffadc44440f3098bf16b) to fix `mypy` errors. Would be better if you could `git pull` remote changes to your branch first before you force push in the future.
> 
> Same thing with the commits pushed by the `pre-commit.ci` bot. Simply pull them in with `git pull`.
> 
> > I changed the default legends to "0", "1" and "2" so that they fit in the box. But now the rgb_label variable can be passed to the method to change the legend. How does that sound?
> 
> Sounds good. 👍

Oh sure. I'll do that in the future 👍.

Reply:
> One more question: do we really need `save_proj_plot()`? Seems like it's a wrapper around calling `fig.savefig()` which can just be left to the user.

I intended to write a counterpart for 'save_plot()'. But I think you are right that it's not necessary.